src/mumble/Log.cpp

```diff
@@ -303,6 +308,20 @@ QString Log::formatChannel(::Channel *c) {
 	return QString::fromLatin1("<a href='channelid://%1/%3' class='log-channel'>%2</a>").arg(c->iId).arg(c->qsName.toHtmlEscaped()).arg(QString::fromLatin1(g.sh->qbaDigest.toBase64()));
 }
 
+void Log::deferLog(Log::MsgType mt, const QString &console, const QString &terse, bool ownMessage, const QString &overrideTTS) {
+	QMutexLocker mLock(&Log::qmDeferredLogs);
+
+	qvDeferredLogs.append(LogMessage(mt, console, terse, ownMessage, overrideTTS));
+}
```

Comment:
Considering that we will only be using `logOrDefer()` when the global `Log` object is not available (and thus the `if` check will only be executed when Mumble is starting), I would move the content of `deferLog()` in `logOrDefer()` and remove the function.

Reply:
good point